src/resources/js/tribe-events.js

```diff
@@ -1681,9 +1681,11 @@ Date.prototype.format = function( mask, utc ) {
 
 		/**
 		 * @function tribe_ical_url
-		 * @desc tribe_ical_url This function adds required params to the ical url. Runs on doc ready, and hooks into 'ajax-success.tribe' also.
+		 * @desc tribe_ical_url This function adds required params to the ical url.
+		 *                       Runs on doc ready, and hooks into 'ajax-success.tribe' also.
+		 * @deprecated TBD
 		 */
-		function tribe_ical_url() {
+		function tribe_ical_url() { // eslint-disable-line no-unused-vars
```

Comment:
This looks good, I have been adding a console.info when deprecating javascript methods. It is something that could be used here too. Example:
		console.info( 'Method deprecated and replaced with virtualAdmin.handleVideoSourceClasses.' ); // eslint-disable-line no-console, max-len